apps/jobs/views.py

```diff
@@ -483,6 +486,18 @@ def change_submission_data_and_visibility(
     if serializer.is_valid():
         serializer.save()
         response_data = serializer.data
+        if (
+            request.FILES.get("submission_input_file")
+            and challenge.is_static_dataset_docker_based_challenge
```

Comment:
@savish28 did you test this API by making a call with `submission_input_file`?

Reply:
@Ram81 Yeah I have checked both the file submission and also the message by making a static code upload challenge and a normal challenge by Postman.